font/src/ft/mod.rs

```diff
@@ -365,12 +365,16 @@ impl FreeTypeRasterizer {
         let hinting = pat.hintstyle().next().unwrap_or(fc::HintStyle::Slight);
         let rgba = pat.rgba().next().unwrap_or(fc::Rgba::Unknown);
 
-        use freetype::face::*;
-
         match (antialias, hinting, rgba) {
-            (false, fc::HintStyle::None, _) => NO_HINTING | MONOCHROME,
-            (false, _, _) => TARGET_MONO | MONOCHROME,
-            (true, fc::HintStyle::None, _) => NO_HINTING | TARGET_NORMAL,
+            (false, fc::HintStyle::None, _) => {
+                freetype::face::LoadFlag::NO_HINTING | freetype::face::LoadFlag::MONOCHROME
+            }
+            (false, _, _) => {
+                freetype::face::LoadFlag::TARGET_MONO | freetype::face::LoadFlag::MONOCHROME
+            }
+            (true, fc::HintStyle::None, _) => {
+                freetype::face::LoadFlag::NO_HINTING | freetype::face::LoadFlag::TARGET_NORMAL
```

Comment:
`freetype::face::LoadFlag` is used a bunch of times. It would probably be appropriate to declare it as in import to reduce code bloat a bit.

Reply:
Sadly, `LoadFlag` is now a `struct`, and all these flags are `const` fields of that struct, so an import doesn't work. Specifically: `use freetype::face::LoadFlag::*` doesn't compile.

Reply:
Oh yeah, that's plenty. Just to shorten the path, importing the members directly is a bit much anyways, since that just obfuscates where they're coming from.

Reply:
Fixed in 0ac281f42ca9567c79d0c03047c4b4b6f460004c